hugegraph-server/hugegraph-dist/src/assembly/travis/test-check-port.sh

```
@@ -0,0 +1,481 @@
+
+# ── download() fallback ───────────────────────────────────────────────────────
+
+section "download() curl fallback (pd/store)"
+
+
+(
+    if [[ ! -f "$PD_UTIL_SH" ]]; then
+        exit 77
+    fi
+    
+    source "$PD_UTIL_SH"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)